docs/deploy.rst

```diff
@@ -0,0 +1,244 @@
+.. _deploy:
+
+
+=========================
+Deploy your first cluster
+=========================
+
+This page shows you how to:
+
+- Create an organization
+- Set a region in Croud (determines where your data is stored)
+- Create a project
+- Check for available CrateDB products in that region
+- Deploy your first CrateDB cluster
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+.. NOTE::
+
+  This document assumes that you're `signed in`_ to your CrateDB Cloud user
+  account.
+
+
+Create an organization
+----------------------
+
+To begin with, let's create an organization. Organizations hold projects
+that let you deploy CrateDB products such as `clusters`_. Once you've
+created an organization, you can invite CrateDB Cloud users to become
+`organization members`_.
+
+Per default, you will be assigned as the administrator of an organization that
+you create.
+
+Give the `croud organizations create`_ command and pass in ``--name`` to specify
```

Comment:
reword. this para essentially prose-ifies the example command. this is too repetitive. instead, say something more like "perform [x action] by running:" you can link the action name to the create command